crates/l2/starknet-core-contract-client/src/lib.rs

```diff
@@ -1,96 +1,65 @@
 use std::sync::Arc;
 
-use crate::clients::StarknetValidityContractClient;
-use clients::StarknetSovereignContractClient;
+use crate::clients::{StarknetCoreContractClient, StarknetCoreContractOverrideClient};
 use starknet_proxy_client::deploy::{deploy_contract_behind_proxy, Error, ProxyVersion};
 use utils::{LocalWalletSignerMiddleware, NO_CONSTRUCTOR_ARG};
 
 pub mod clients;
 pub mod interfaces;
 
-const STARKNET_SOVEREIGN: &str = include_str!("artifacts/StarknetSovereign.json");
-const STARKNET: &str = include_str!("artifacts/Starknet.json");
+const STARKNET_CORE_CONTRACT: &str = include_str!("../../../../artifacts/cairo-lang/Starknet.json");
+const STARKNET_CORE_CONTRACT_OVERRIDE: &str =
+    include_str!("../../../../artifacts/StarknetOverride.json");
```

Comment:
we were planning to call this DevCoreContract right?

Reply:
renamed